frontend/package.json

```diff
@@ -152,7 +152,6 @@
     "@patternfly/react-topology": "5.3.0",
     "@patternfly/react-user-feedback": "5.0.0",
     "@patternfly/react-virtualized-extension": "5.1.0",
-    "@prometheus-io/codemirror-promql": "^0.37.0",
```

Comment:
this was supposed to be removed in https://github.com/openshift/console/pull/14442